sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<h2>View SageMaker Debugger Reports  </h2><p>SageMaker Debugger generates profiling and training reports through a pair of processing jobs that run concurrent to the training job. The code below will download the outputs from the Debugger report output S3 URI to your current Jupyter working directory for easier viewing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>